lua/neoclip/handlers.lua

```diff
@@ -94,4 +99,24 @@ M.delete = function(typ, entry)
     storage.delete(typ, entry)
 end
 
+M.handle_system_yank = function ()
+    if neoclip.stopped then
+        return
+    end
+    if not settings.enable_system_clipboard then
```

Comment:
I don't think this setting is needed anymore since a user just wouldn't trigger this function